index.html

```diff
@@ -20,97 +20,97 @@ <h3>Lopper</h3>
         </ul>
     </header>
     <main>
-        <div>
-            <h1>About Us</h1>
-            <p>WE BUILD GREAT BUSINESS</p>
-        </div>
-        <section>
-            <p>WHAT WE DO</p>
-            <h2>Apparently we had reached a great height in the 
+        <section class="main_title">
+            <h1 class="main_text">About Us</h1>
+            <p class="main_paragraph">WE BUILD GREAT BUSINESS</p>
+        </section>
+        <section class="main_about">
+            <p class="about_subtitle">WHAT WE DO</p>
+            <h2 class="about_text">Apparently we had reached a great height in the 
```

Comment:
Os textos abaixo do WHAT WE DO são como uma descrição da seção, seria melhor ser um `<p>`